spring-cloud-deployer-local/src/main/java/org/springframework/cloud/deployer/spi/local/AbstractLocalDeployerSupport.java

```diff
@@ -50,12 +52,15 @@
  * @author Ilayaperumal Gopinathan
  * @author Thomas Risberg
  * @author Oleg Zhurakousky
+ * @author Vinicius Carvalho
  */
 public abstract class AbstractLocalDeployerSupport {
 
 	public static final String USE_SPRING_APPLICATION_JSON_KEY =
 			LocalDeployerProperties.PREFIX + ".use-spring-application-json";
 
+	public static final String SPRING_APPLICATION_JSON = "SPRING_APPLICATION_JSON";
```

Comment:
I know this isn't a short term option, but since Boot doesn't expose this as a constant that we can consume, we may want to consider moving this to the SPI project so it's consistent across the deployers.

Reply:
100% agreeing with you, but since touching the SPI was something a bit more complicated, I left it behind, just moved to a constant to avoid typos seemed logical

Reply:
Actually...change this comment.  I asked Boot to create a constant for this value and they have.  Once available, that would be the best route IMHO.